src/NUnitFramework/framework/nunit.framework-netcore/nunit.framework-netcore.csproj

```diff
@@ -0,0 +1,1062 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This is another odd thing about DNX. If the project is in the same directory as our other projects, Visual Studio uses the project.json file that gets added to DNX projects for all the projects in that directory causing all the other projects to fail to build. As a workaround, I put the projects in subfolders.